scripts/cron_eval/eval.py

```
@@ -716,6 +745,16 @@ def main() -> int:
         print(f"  cron_persistence: {'PASS' if ok else 'FAIL'}")
         return 0 if ok else 1
 
+    # Full run: reset broker state first so each invocation starts from a
+    # deterministic baseline (clears overrides left by the previous run).
+    # Also remove a stale snapshot so --verify-persistence won't pick up
+    # a result from a different broker boot.
+    print(f"[cron-eval] resetting scheduler state before run…")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)